zebedee-cms/src/main/java/com/github/onsdigital/zebedee/dataset/api/DatasetAPIClient.java

```diff
@@ -98,13 +100,56 @@ public Dataset getDataset(String datasetID) throws IOException, DatasetNotFoundE
     }
 
     /**
+     * Get a particular version of a dataset.
+     * @param datasetID
+     * @param edition
+     * @param version
+     * @return
+     * @throws IOException
+     * @throws DatasetAPIException
+     */
+    @Override
+    public DatasetVersion getDatasetVersion(String datasetID, String edition, String version) throws IOException, DatasetAPIException {
+
+        if (StringUtils.isEmpty(datasetID) || StringUtils.isEmpty(edition)) {
+            throw new BadRequestException("A dataset ID, edition, and version must be provided.");
+        }
+
+        String path = "/datasets/" + datasetID + "/editions/" + edition + "/versions/" + version ;
```

Comment:
Is it worth using a URL builder?

Reply:
Good point - I have refactored to use the builder.